packages/tables/backend/sirius-components-tables/src/main/java/org/eclipse/sirius/components/tables/components/TableComponent.java

```diff
@@ -56,6 +58,16 @@ public Element render() {
         TableRenderingCache cache = new TableRenderingCache();
         ITableElementRequestor tableElementRequestor = new TableElementRequestor();
 
+        AtomicReference<String> globalFilter = new AtomicReference<>("");
+        if (this.props.globalFilter() == null) {
+            optionalPreviousTable.ifPresent(previousTable -> {
+                globalFilter.set(previousTable.getGlobalFilter());
+                variableManager.put(TableRenderer.GLOBAL_FILTER_DATA, previousTable.getGlobalFilter());
+            });
+        } else {
+            globalFilter.set(this.props.globalFilter());
```

Comment:
If the globalFilter given as a props is not null then it is not added to the variable manager as a variable and thus it does not work.